scripts/create_network.py

```diff
@@ -42,10 +36,156 @@ def create_network():
     return n
 
 
-def add_buses_to_network(n, number_microgrids):
-    # Add buses to the network based on the number of microgrids
-    for i in range(number_microgrids):
-        n.madd("Bus", [f"bus{i+1}"], carrier="AC", v_nom=0.220)
+def create_microgrid_network(
+    n, input_file, number_microgrids, voltage_level, line_type
+):
+    """
+    Creates local microgrid networks within the PyPSA network. The local microgrid networks are distribution networks created based on
+    the buildings data, stored in "resources/buildings/microgrids_buildings.geojson". Then the buses are connected together through lines
+    according to the output of a Delaunay Triangulation.
+    """
+    # Load the GeoJSON file
+
+    with open(input_file) as f:
+        data = json.load(f)
+
+    # Keep track of the bus coordinates and microgrid IDs
+    bus_coords = set()
+    number_microgrids = len(number_microgrids.keys())
+    microgrid_ids = [f"microgrid_{i+1}" for i in range(number_microgrids)]
```

Comment:
I don't really like the "microgrid_ids" couldn't we use the name of the microgrid as specified in the config?
It could be the name/index of the geodataframe.